console-timber-tree/src/main/java/com/jraska/console/timber/ConsoleTree.java

```diff
@@ -39,6 +44,8 @@
 
   private final int[] priorityColorMapping;
 
+  private final String timeFormat;
```

Comment:
Could we please change this to have `DateFormat` instead of `String`?

It will get more flexibility to user, will move the responsibility of its creation out of Console and also will prevent the unfortunate situation of providing invalid format and crashing in he middle of logging.